tests/e2e/specs/workspace.cy.js

```diff
@@ -162,7 +166,34 @@ describe('Workspace view and component/widget', () => {
 
     // Navigate back to original workflow
     cy.visit('/#/workspace/one')
-    expectSplitPane()
+    expectRememberedLayout()
+
+    cy.window().then((win) => {
+      if (win.caches) {
+        // Test page refresh
+        cy.reload()
+        expectRememberedLayout()
+      } else {
+        cy.log('Cache API not supported; skipping page refresh test.')
+      }
+    })
+  })
+
+  it('Resets layout', () => {
```

Comment:
Could you add this to another test to save the setup time?

Reply:
There are downsides to overloading tests, as if something changes in the middle it might lead to invalidating the rest of the test, and failures can be more difficult to isolate